invenio_stats/aggregations.py

```diff
@@ -361,17 +395,27 @@ def agg_iter(self, dt):
                     else:
                         aggregation_data[destination] = source(doc, aggregation_data)
 
-                index_name = "stats-{0}-{1}".format(
-                    self.event, interval_date.strftime(self.index_name_suffix)
+                index_name = prefix_index(
+                    "stats-{0}-{1}".format(
+                        self.event, interval_date.strftime(self.index_name_suffix)
+                    )
                 )
 
                 yield {
                     "_id": "{0}-{1}".format(
                         aggregation["key"], interval_date.strftime(self.doc_id_suffix)
                     ),
-                    "_index": prefix_index(index_name),
+                    "_index": index_name,
                     "_source": aggregation_data,
                 }
+                if "parent_recid" in doc:
+                    yield {
+                        "_index": prefix_index("buffer_stats_to_reindex"),
```

Comment:
Is this a new index?

Reply:
Yep, new index with the entries that have been modified ny the aggregations and that should be reindexed